lib/suspenders/adapters/heroku.rb

```diff
@@ -0,0 +1,67 @@
+module Suspenders
+  module Adapters
+    class Heroku
+      def initialize(app_builder)
+        @app_builder = app_builder
+      end
+
+      def set_heroku_remotes
+        remotes = <<-SHELL.strip_heredoc
+
+          # Set up the staging and production apps.
+          #{join_heroku_app('staging')}
```

Comment:
Given this is a refactor, it makes sense to me to rename variables or methods that can get better names. :+1: